hstream-http-server/src/HStream/HTTP/Server/Stream.hs

```diff
@@ -81,7 +81,7 @@ listStreamsHandler hClient = liftIO $ do
 deleteStreamHandler :: Client -> String -> Handler Bool
 deleteStreamHandler hClient sName = liftIO $ do
   HStreamApi{..} <- hstreamApiClient hClient
-  let deleteStreamRequest = DeleteStreamRequest { deleteStreamRequestStreamName = TL.pack sName }
+  let deleteStreamRequest = DeleteStreamRequest { deleteStreamRequestStreamName = TL.pack sName, deleteStreamRequestIgnoreNonExist = True }
```

Comment:
`deleteStreamRequestIgnoreNonExist` should be False?

Reply:
Do user need to know that the stream was existed?